zimbra/src/main/java/fr/openent/zimbra/helper/ServiceManager.java

```diff
@@ -109,7 +113,7 @@ private ServiceManager(Vertx vertx, EventBus eb, String pathPrefix) {
         this.signatureService = new SignatureService(userService, soapService);
         this.messageService = new MessageService(soapService, folderService,
                 dbMailServiceApp, userService, synchroUserService);
-        this.attachmentService = new AttachmentService(soapService, messageService, vertx, rawConfig);
+        this.attachmentService = new AttachmentService(soapService, messageService, vertx, rawConfig, webClient);
```

Comment:
t'es obligé de le donner en param ? tu peux pas l'init dans le attachment service?

Reply:
ça j'ai vu avec Jin parce que dans la documentation vertx, c'est marqué qu'il vaut mieux utilisé un seul webClient (on le crée une fois et ensuite on le réutilise) : 

Important | 
In most cases, a Web Client should be created once on application startup and then reused. Otherwise you lose a lot of benefits such as connection pooling and may leak resources if instances are not closed properly.

https://vertx.io/docs/vertx-web-client/java/